substrate/frame/support/src/traits/tokens/fungible/hold.rs

```diff
@@ -449,6 +449,16 @@ pub trait Balanced<AccountId>: super::Balanced<AccountId> + Unbalanced<AccountId
 		Self::done_slash(reason, who, decrease);
 		(credit, amount.saturating_sub(decrease))
 	}
+}
 
-	fn done_slash(_reason: &Self::Reason, _who: &AccountId, _amount: Self::Balance) {}
+pub trait DoneSlash<Reason, AccountId, Balance> {
```

Comment:
No undocumented pub items please: https://github.com/paritytech/polkadot-sdk/blob/master/docs/contributor/DOCUMENTATION_GUIDELINES.md#what-to-document